docs/about.md

```diff
@@ -0,0 +1,80 @@
+# About Heptio Ark
+
+Heptio Ark provides customizable degrees of recovery for all Kubernetes objects (Pods, Deployments, Jobs, Custom Resource Definitions, etc.), as well as for persistent volumes. This recovery can be cluster-wide, or fine-tuned according to object type, namespace, or labels.
+
+Ark is ideal for the disaster recovery use case, as well as for snapshotting your application state, prior to performing system operations on your cluster (e.g. upgrades).
+
+## Features
+
+Ark provides the following operations:
+
+* On-demand backups
+* Scheduled backups
+* Restores
+
+Each operation is a custom resource, defined with a Kubernetes [Custom Resource Definition (CRD)][20] and stored in [etcd][22]. An additional custom resource, Config, specifies cloud provider settings. These resources are handled by [custom controllers][21] when their corresponding requests are submitted to the Kubernetes API server.
+
+Each controller watches its custom resource for API requests (Ark operations), performs validations, and handles the logic for interacting with the cloud provider API -- for example, managing object storage and persistent volumes.
+
+### On-demand backups
+
+The **backup** operation:
+
+1. Uploads a tarball of copied Kubernetes objects into cloud object storage.
+
+1. Calls the cloud provider API to make disk snapshots of persistent volumes, if specified.
```

Comment:
I think it's been mentioned elsewhere, but things like Minio and Rook aren't necessarily cloud providers, though Ark could operate against them. Maybe refer to 'block storage API' instead of 'cloud provider' here.

Reply:
Let's hold off changing any language here until we do backup targets. This is fine for now.